cirq/ops/linear_combinations.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2018 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Any, Dict, Mapping, Optional, Tuple
+
+import numpy as np
+
+from cirq import protocols, value
+from cirq.ops import raw_types
+
+
+class LinearCombinationOfGates(value.LinearDict[raw_types.Gate]):
+    """Represents linear operator defined by a linear combination of gates.
+
+    Note that the operator may not be unitary or even normal. Examples of
```

Comment:
This doesn't explain what the object is; it's just repeating the same words.

Say explicitly that this thing is supposed to be the sum of the matrices defining the gates.

Reply:
Clarified.